packages/techdocs-cli-embedded-app/src/components/TechDocsPage/TechDocsPage.tsx

```diff
@@ -49,15 +49,15 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
 }));
 
-enum Themes {
-  LIGHT = 'light',
-  DARK = 'dark',
-}
+const Themes = {
+  LIGHT: 'light',
+  DARK: 'dark',
+} as const;
 
 export const TechDocsThemeToggle = () => {
   const appThemeApi = useApi(appThemeApiRef);
   const classes = useStyles();
-  const [theme, setTheme] = useState<Themes>(
+  const [theme, setTheme] = useState<(typeof Themes)[keyof typeof Themes]>(
```

Comment:
Something like this could be just `'light' | 'dark'` instead, but no super strong opinion on that